Collector/MigrationsCollector.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/*
+ * This file is part of the Doctrine MigrationsBundle
+ *
+ * The code was originally distributed inside the Symfony framework.
+ *
+ * (c) Michael Simonson <mike@simonson.be>
```

Comment:
the copyright header should stay the same than in other files of the project (this is not tracking the author of the file)